src/include/basil.h

```diff
@@ -691,7 +792,13 @@ typedef struct basil_node {
 	basil_node_arch_t arch;
 	basil_node_state_t state;
 	basil_node_role_t role;
+	unsigned int numsockets;
+	unsigned int numdies;	/* dies = segments */
```

Comment:
The term "segment" was used to represent a portion of a processor that was logically independent of other segments. The term was meant to be vendor agnostic so that processor manufacturers could use whatever nomenclature they felt appropriate without BASIL having to change. If the use of "dies" is something introduced in more recent versions of BASIL, then feel free to use it. If not, be very careful when adding new component names.